idaes/apps/caprese/examples/cstr_rodrigo/nmpc_main.py

```diff
@@ -0,0 +1,179 @@
+##############################################################################
+# Institute for the Design of Advanced Energy Systems Process Systems
+# Engineering Framework (IDAES PSE Framework) Copyright (c) 2018-2019, by the
+# software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia
+# University Research Corporation, et al. All rights reserved.
+#
+# Please see the files COPYRIGHT.txt and LICENSE.txt for full copyright and
+# license information, respectively. Both files are also available online
+# at the URL "https://github.com/IDAES/idaes-pse".
+##############################################################################
+"""
+Example for Caprese's module for NMPC.
+Main script for running the example.
+"""
+import random
+from idaes.apps.caprese.nmpc import NMPCSim
+from idaes.apps.caprese.util import apply_noise_with_bounds
+from pyomo.environ import SolverFactory
+from pyomo.dae.initialization import solve_consistent_initial_conditions
+import idaes.logger as idaeslog
+from cstr_rodrigo_model import make_model
```

Comment:
This import causes an ImportError if this file is run or imported from outside this directory. Can you change this to be an absolute import from the `idaes` package, i.e.
```python
from idaes.apps.caprese.examples.cstr_rodrigo_model import make_model
```

Reply:
No problem! Thanks for catching that!